gemrb/plugins/CREImporter/CREImporter.cpp

```diff
@@ -567,8 +566,8 @@ void CREImporter::WriteChrHeader(DataStream *stream, const Actor *act)
 	}
 	stream->Write( Signature, 8);
 	memset( Signature,0,sizeof(Signature));
-	memset( name,0,sizeof(name));
-	strlcpy(name, act->GetName(0), sizeof(name));
+	ieVariable name = act->GetNameAsVariable(0);
+	strlwr(name);
```

Comment:
This forced lowercasing is bad, most names will have at least one capital letter.

Reply:
I was in blind autopilot mode thinking `strlcpy` copied a string and lowercased it :)